modules/src/coinflip.py

```diff
@@ -0,0 +1,18 @@
+import random
+from templates.text import TextTemplate
+
+def process(input, entities):
+    output = {}
+    try:
```

Comment:
no need of try/catch as there is no api call, external file loading, etc. that can fail